doc/tutorials/setup_openstack_libvert.rst

```diff
@@ -0,0 +1,283 @@
+.. _setup_openstack_libvert:
+
+Configure ACRN using OpenStack and libvert
+##########################################
+
+Introduction
+************
+
+This document provides instructions for setting up libert to configure ACRN. We use OpenStack to use libvert. Install OpenStack in a container to avoid crashing your system and to take advantage of easy snapshots/restores so that you can quickly roll badk your system in the event of setup failure. Install OpenStack directly on Ubuntu **only** if you have a dedicated testing machine. This setup utilizes LXC/LXD on Ubuntu 16.04/18.04.
+
+Install ACRN
+************
+
+#. Install ACRN using Ubuntu 16.04 or 18.04 as its Service VM. Refer to :ref:`Ubuntu Service OS`.
+
+#. Make acrn-kernel from the attached configuration:
+   ``kernel_config_uefi_sos_openstack``
```

Comment:
@peterfang @dbkinder @gvancuts We will need to supply the patch here as opposed to providing a hyperlink (@dbkinder can help with this).

Reply:
Thanks!